tests/conftest.py

```diff
@@ -732,7 +732,7 @@ def __init__(
                     set_default_torch_num_threads(default_torch_num_threads))
 
         if not kwargs.get("compilation_config", None):
-            kwargs["compilation_config"] = {"cudagraph_capture_sizes": [8]}
```

Comment:
better to leave some comments here saying that `4` is not an arbitrary choice, in case people change it further in the future. @tdoublep